lms/urls.py

```diff
@@ -26,9 +26,6 @@
     url(r'^change_email$', 'student.views.change_email_request', name="change_email"),
     url(r'^email_confirm/(?P<key>[^/]*)$', 'student.views.confirm_email_change'),
     url(r'^change_name$', 'student.views.change_name_request', name="change_name"),
-    url(r'^accept_name_change$', 'student.views.accept_name_change'),
```

Comment:
We talked about this offline, but for the sake of anyone following this PR, I think this is fine. These are old, prototype-days endpoints that were never advertised as APIs. As long as we shift the front end feature that uses it over to use the new thing, I don't think there's a problem. I would suggest a courtesy email to edx-code explaining this.